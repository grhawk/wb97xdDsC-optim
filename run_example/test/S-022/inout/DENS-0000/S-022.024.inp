 $DFT NRAD=24 NLEB=110 DDSC=.t. $END
 $BASIS GBASIS=N31 NGAUSS=6 $END
 $CONTRL MAXIT=200 RUNTYP=ENERGY EXETYP=RUN SCFTYP=ROHF DFTTYP=wB97X 
ICHARG=0 MULT=1 $END
 $SCF DIRSCF=.t. $END
 $SYSTEM MWORDS=8 $END
 $DATA
 Should be setted!
 C1
 O     8.00    -1.551007    -0.114520     0.000000
 H     1.00    -1.934259     0.762503     0.000000
 H     1.00    -0.599677     0.040712     0.000000
 $END