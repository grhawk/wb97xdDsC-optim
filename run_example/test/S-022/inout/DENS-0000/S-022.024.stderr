/var/spool/slurm/job10168/slurm_script: line 9: module: command not found
